ui/components/app/selected-account/selected-account-component.test.js

```diff
@@ -52,13 +52,6 @@ jest.mock('../../../selectors', () => {
     getAccountType: mockGetAccountType,
     getSelectedInternalAccount: mockGetSelectedAccount,
     getCurrentChainId: jest.fn(() => '0x5'),
-    getSelectedNetworkClientId: jest.fn(() => 'goerli'),
-    getNetworkConfigurationsByChainId: jest.fn(() => ({
-      '0x5': {
-        chainId: '0x5',
-        rpcEndpoints: [{ networkClientId: 'goerli' }],
-      },
-    })),
```

Comment:
These functions were moved, and the tests pass without these being mocked. so I removed them.